src/guides/v2.3/graphql/mutations/send-email-to-friend.md

```diff
@@ -127,3 +127,17 @@ Attribute |  Data Type | Description
 `email` | String | The email address of the sender
 `message` | String | The text of the message
 `name` | String | The name of the sender
+
+## Errors
+
```

Comment:
Listing the click paths seems overly specific in this section. Add a line similar to the following:

`Some errors occur because the **Email a friend** feature is not fully enabled. Go to **Stores** > **Configuration** > **Catalog** > **Email to a friend** in the Admin to adjust the settings.`

Reply:
**done**

Reply:
**done**